project1/styles.css

```diff
@@ -1,5 +1,67 @@
+body {
+    background-color: hsl(254, 0%, 100%);
+    border: 3px;
+}
+
+figure {
+    float: right;
+    margin-right: 40px;
+    padding: 30px;
+
+
+}
+
+iframe {
+    margin: 60px;
+}
+
 img {
-    max-width: 100%
+    max-width: 100%;
 }
 
-/* comments look like this */
+figcaption {
+    color: hsl(0, 0%, 40%);
+    font-family: 'amiri', serif;
+}
+
+h1 {
+    color: hsl(299, 32%, 60%);
+    text-align: center;
+    font-family: 'amiri', serif;
+    font-size: 48px;
+    font-weight: normal;
+    margin: 30px;
+}
+
+h2 {
```

Comment:
You could combine some of the shared styles for your headings, like so:

```
h1,
h2 {
    font-family: 'amiri', serif;
    font-weight: normal;
    margin: 30px;
    text-align: center;
}

h1 {
    color: hsl(299, 32%, 60%);
    font-size: 48px;
}

h2 {
    color: hsl(0, 0%, 40%);
    font-size: 21px;
}
```

This way, it is easy to tell at a glance, what they have in common and how they are different 👍🏻